Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -478,6 +485,22 @@ private void TrySetParam(DB.Element elem, BuiltInParameter bip, double value, st
       }
     }
 
+    /// <summary>
+    /// Queries a Revit Document for phases by the given name.
+    /// </summary>
+    /// <param name="document"></param>
+    /// <param name="phaseName">The name of the Phase</param>
+    /// <returns>the phase which has the same name. null if none or multiple phases were found.</returns>
+    private Element GetRevitPhase(DB.Document document, string phaseName )
+    {
+      var phases = new FilteredElementCollector(document).OfCategory(BuiltInCategory.OST_Phases)
```

Comment:
We could cache the list of Phases the first time this function is run to avoid having to fetch them for each element, this could speed conversion quite a bit on large models

Reply:
Definitely! Where would we store those values?

Reply:
For consistency, we could have it next to the cached levels: https://github.com/specklesystems/speckle-sharp/blob/main/Objects/Converters/ConverterRevit/ConverterRevitShared/ConverterRevit.cs#L72

The converter is re-instantiated for each conversion so no need to clear the list ;)

Reply:
Done